acl/validation.go

```diff
@@ -40,10 +45,21 @@ func IsValidNodeIdentityName(name string) bool {
 	return validNodeIdentityName.MatchString(name)
 }
 
-// IsValidPolicyName returns true if the provided name can be used as an
-// ACLPolicy Name.
-func IsValidPolicyName(name string) bool {
-	return validPolicyName.MatchString(name)
+// IsValidPolicyName returns nil if the provided name can be used as an
+// ACLPolicy Name otherwise a useful error is returned.
+func IsValidPolicyName(name string) error {
```

Comment:
nit: I might rename this to maybe `ValidatePolicyName` because it returns an error (whereas `IsValid...` indicates that it returns a bool). What do you think?

Reply:
Changed!